packages/openhim-core-api/package.json

```
@@ -0,0 +1,52 @@
+{
+  "name": "@jembi/openhim-core",
```

Fixing the package name to match what we have in root-config index.ejs

```suggestion
  "name": "@jembi/openhim-core-api",
```